deploy/scripts/start-service.sh

```diff
@@ -49,7 +49,7 @@ docker run \
     --network openregisters \
     --log-driver=fluentd \
     --log-opt fluentd-async-connect=true \
-    openjdk:8-jre-alpine \
```

Comment:
I put something in the git commit that introduced this:

> Upgrading to the latest version (8u141) as well as moving to using
images based off Debian base images. It seems that Debian have more
timely responses to security issues with packages. Version 8u141 has
been available for >7 days, has been in Debian Stretch >2 days but still
hasn't landed in a release version of Alpine. It has been available in
Alpine Edge for >7 days but not in an actual release.